iOS/ExposureNotification/source/ExposureNotification.csproj

```diff
@@ -43,5 +42,4 @@
     <ObjcBindingApiDefinition Include="ApiDefinition.cs" Condition=" '$(EnableDefaultCompileItems)' == 'true'" />
   </ItemGroup>
 
-  <Import Project="$(MSBuildSDKExtrasTargets)" Condition="Exists('$(MSBuildSDKExtrasTargets)')" />
```

Comment:
Is this meant to be done? is it not needed anymore?

Reply:
Not any more. The new Sdk attribute at the top does it for us :)